packages/manager/modules/pci/src/projects/creating/routing.js

```diff
@@ -6,7 +6,7 @@ import { GUIDE_URLS } from './constants';
 
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('pci.projects.creating', {
-    url: '/creating/:orderId/:voucherCode',
+    url: '/creating/:orderId/:voucherCode/:isDiscoveryProject',
```

Comment:
**question**: is there a more secure way we can define this boolean using the api instead of passing it to the url ?

I think we could with the orderId : /me/order/{orderId}/details/{detailId} you get the domain witch is a PCI service name. Using this service name on /cloud/project/{serviceName} you'll find the planCode

Reply:
The only use of this attribute is to filter the display of two minor UI components and is not used to perform any backend actions. So, don't see a security impact.